import sqlite3
from sqlite3 import Error

def ConexaoBanco(caminho):
    caminho ="C:\Users\joaog\OneDrive\Desktop\Banco de dados"
    